addons/mail/static/src/core/web/messaging_menu.js

```diff
@@ -80,6 +86,19 @@ export class MessagingMenu extends Component {
         );
     }
 
+    get installationRequest() {
+        return {
+            body: _t("Come here often? Install Odoo on your device!"),
+            displayName: sprintf(_t("%s has a suggestion"), this.store.odoobot.name),
```

Comment:
I don't think you need to wrap the translation in `sprintf()` call ; just pass the argument to the `_t()` function.